kratos/geometries/tetrahedra_3d_4.h

```diff
@@ -681,7 +681,11 @@ template<class TPointType> class Tetrahedra3D4 : public Geometry<TPointType>
       double se = (e[0]*e[0]) + (e[1]*e[1]) + (e[2]*e[2]);
       double sf = (f[0]*f[0]) + (f[1]*f[1]) + (f[2]*f[2]);
 
-      return normFactor * std::pow(3*Volume(), 2.0 / 3.0) / (sa + sb + sc + sd + se + sf);
+      double vol = Volume();
+      double absVol = std::abs(Volume());
+      double isInv = vol < 0.0 ? -1.0 : 1.0;
+
+      return normFactor * isInv * std::pow(3*absVol, 2.0 / 3.0) / (sa + sb + sc + sd + se + sf);
```

Comment:
Actually I have a concern here. Is there any way to do this in a more clean way?